src/main/java/module/playerOverview/SpielerTrainingsVergleichsPanel.java

```diff
@@ -52,32 +55,36 @@ public SpielerTrainingsVergleichsPanel() {
     }
 
     //~ Methods ------------------------------------------------------------------------------------
-    public static boolean isVergleichsMarkierung() {
+    public static boolean isDevelopmentStageSelected() {
         return vergleichsMarkierung;
     }
 
     /**
      * Gibt die Vergleichsspieler zurück
      */
-    public static List<Player> getVergleichsPlayer() {
+    public static List<Player> getSelectedPlayerDevelopmentStage() {
         return vergleichsPlayer;
     }
 
     public final void actionPerformed(java.awt.event.ActionEvent actionEvent) {
-        final Object[] hrfs = m_jlHRFs.getSelectedValues();
+        final var hrfs = m_jlHRFs.getSelectedValuesList();
         StringBuilder text = new StringBuilder(HOVerwaltung.instance().getLanguageString("ls.button.delete"));
 
-        if (hrfs.length > 1) {
-            text.append(" (").append(hrfs.length).append(" Files) : ");
+        if (hrfs.size() > 1) {
+            text.append(" (").append(hrfs.size()).append(" Files) : ");
         } else {
             text.append(": ");
         }
 
-        for (int i = 0; (i < hrfs.length) && (i < 11); i++) {
-            text.append("\n").append(hrfs[i].toString());
-
-            if (i == 10) {
-                text.append("\n ... ");
+        int i=0;
+        for (var hrf : hrfs){
+            text.append("\n");
+            if ( i++ < 11){
```

Comment:
Could you please extract that magic number?

Reply:
changed this - thanks